site/src/pages/components/combobox.explainer.mdx

```diff
@@ -0,0 +1,277 @@
+
+The `<select>` element, for instance, while functional, lacks flexibility. It offers limited customization, which often leads developers to craft their own solutions. These custom implementations, while visually appealing, often come at the cost of performance, reliability, and most importantly, accessibility.
+
+Recognizing these limitations, the [`<selectlist>`](https://open-ui.org/components/selectlist/) was introduced. This element builds on the foundation of the `<select>` but provides greater flexibility. The `<selectlist>` lets developers create a button-triggered dropdown with customizable appearance and behavior. Paired with `<listbox>`, and inclusive of `<option>` and `<optgroup>`, it offers a modern, adaptable, yet accessible web control.
+
+Right now, many websites have their own type of **'combobox'**, but they're all a bit different. We often see a mix of `<input>` fields and `<datalist>` elements trying to do the job, but it's not quite perfect. By creating a standard **`<combobox>`**, we're aiming to simplify things. 
+
+Further highlighting the need for such a solution, a recent [Twitter poll](https://twitter.com/openuicg/status/1717345237889351911) run by the Open UI Community Group underscored the widespread demand for a standardized ``combobox``
+<Image
+  src="/images/combobox/combobox-twitterpoll.png"
+  alt="Combobox twitter poll results"
+/>
+
+
+This proposal for `<combobox>` builds upon the strengths of `<selectlist>` and `<listbox>`. It aims to offer a control that is versatile for developers, intuitive for users, and, above all, accessible to all.
+
+By standardizing such elements, we hope to bridge the gap between design demands and the need for performant, reliable, and accessible web controls. 
+
+### Scope of this document
+
+This document aims to provide a complete view of combo-box utilisation across the web to inform a potential proposal within Open UI. The goal of this document is to propose a very basic anatomy to simplify the initial shipment of `<combobox>` (or some other name) in the web platform. It will heavily build upon `selectlist` but provide a better accessible and extensible solution than that of an `input` bound with `datalist`.
+
+
+### Out of scope of this document
+
+This document will not be attempting some of the complex scenarios found across the web that leverage various ways in which to represent multi-selected options. Additionally, it does not intend to cover the addition of options that are not available in a `<listbox>`. It will not cover multi-select but will leverage the resolutions defined by `selectlist`.
+
+A good analogy for what this document will result in looking like is the [Tabs research document.](https://open-ui.org/components/tabs.research.parts/)
+
+## Recommended anatomy for version 1
+
+```
+<combobox>
+   <input type=selectlist>
```

Comment:
Currently `popovertarget` (which this effectively uses under the hood) only supports buttons. We resolved in #839 that this should be extended to anchors as well. @lukewarlow raised a few weeks back that `input` should be added. This is a compelling reason for `input` to be added to the list.

Here is a jsbin that shows what is currently supported vs what you're proposing: https://jsbin.com/cedepubepe/edit?html,css,output

I recommend adding a "Question" (see class [here](https://github.com/openui/open-ui/blob/b87d781c1ff9df0d04a93e3107ab8c98da47f5fc/site/src/styles/spec.css#L14) ) that this anatomy would require input to be supported by invokertarget.

Reply:
Fwiw combobox was exactly the usecase where I thought invoke (or I think probably actually interest) target would be useful on an input.

Having said that I'm not convinced input is the right call here (discuss separately of course not wanting to block this) due to its parser limitations that I don't forsee us getting past.